### DISCUSSION TITLE
How to use font paths with a Pango command?

### DISCUSSION THREAD

**Discussion by TheCraig** [ImageMagick/ImageMagick#3909] [Help]
Hi, I would ideally like to call fonts along with a Pango command just using the font path rather than installing the fonts?  Is this possible?

I have tried calling the font within the IM command: 

`/usr/bin/convert  \( -gravity center -size 1280x280  -background none   -font /home/server/fonts/Oswald:400.ttf -density 72   -pointsize 52   -fill "#ffffff"  pango:"<span size='72000'>TESTING PANGO</span><i>MORE PANGO</i>"   \)   exportimage.png`

And have tried the font call within Pango:

`/usr/bin/convert  \( -gravity center -size 1280x280  -background none   pango:"<span font_family='/home/server/fonts/Oswald:400.ttf ' size='72000'>TESTING PANGO</span>"   \)     exportimage.png`

Is there a way to do this?

Thanks!
Craig

**Comment by urban-warrior** [comment]
See https://legacy.imagemagick.org/Usage/text/#pango_markup.    You will need to control the fonts and paths within the Pango Markup Language.
